src/hooks/use-url-routing.test.js

```diff
@@ -0,0 +1,308 @@
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import * as history from 'history';
```

Comment:
Unsure how to test the `stateFromUrl` and `onHistoryChange` functions in `use-url-routing` but the external APIs are tested.